doc/sphinx/source/recipes/recipe_esacci_oc.rst

```diff
@@ -0,0 +1,81 @@
+.. _recipes_<mynewrecipe>:
+   
+Ocean chlorophyll in ESMs compared to ESA-CCI observations.
+===========================================================
+
+Overview
+--------
+
+This recipe compares monthly surface chlorophyll from CMIP models to ESA CCI ocean colour chlorophyll. The observations are the merged sensor geographic monthly L3S chlor_a data Sathyendranath et al (2019). Multiple models and different observational versions can be used by the script.
+
+The recipe_esacci_oc.yml produces an image showing four maps. Each of these four maps show latitude vs longitude and the cube value is used as the colour scale. The four plots are: ESA-CCI-OC v5.0 chlorophyll, the CMIP6 model, the bias model-observation and the ratio model/ observations. The script also produces a scatter plot for all coordinates with the model on the x-axis and the observations on the y axis and a line of best fit with the parameter values given in the panel.
+
+
+Available recipes and diagnostics
+---------------------------------
+
+Recipes are stored in esmvaltool/recipes/
+
+    * recipe_esacci_oc.yml
+
+Diagnostics are stored in esmvaltool/diag_scripts/<mynewdiag>/
+
+    * <mynewdiag.py/.ncl/.r>: one line scription
+    * diagnostic_model_vs_obs.py
+
+
+User settings in recipe
+-----------------------
+
+#. Script <mynewdiag.py/.ncl/.r>
+
+   *Required settings for script*
+
+   * xxx: zzz
+
+   *Optional settings for script*
+
+   *Required settings for variables*
+
+   *Optional settings for variables*
+
+   *Required settings for preprocessor*
+
+   *Optional settings for preprocessor*
+
+   *Color tables*
+
+   * list required color tables (if any) here
```

Comment:
This should be populated with the corresponding variables or removed if not needed.